plugins/score-card/src/api/index.ts

```diff
@@ -13,7 +13,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-export type { SystemScore, SystemScoreArea, SystemScoreEntry } from './types';
+export type { EntityScore, EntityScoreArea, EntityScoreEntry, SystemScore, SystemScoreArea, SystemScoreEntry } from './types';
```

Comment:
won't this break legacy apiClients that still works with SystemScore? I mean not exporting SystemScore? Whole /api is exported from the plugin...